docs/first-pr.md

```diff
@@ -573,21 +573,17 @@ in doubt, ask!
 
 Once you've gotten an LGTM from who you think is the right person, don't be
 afraid to merge. The git revert command exists for a reason. You can expect to
-revert at least PR that you land in your first three months.
-
-You should always use the big green "Merge pull request" button in GitHub to
-merge  your code or its equivalent in the Reviewable interface. This limits your
-exposure by ensuring you don't accidentally merge a commit with obviously an
-obviously broken build. Once your commit lands on master, TeamCity will do
-another test run; this can occasionally fail for legitimate reasons due to merge
-skew.
-
-Make sure to use the default "Create a merge commit" mode when you hit the merge
-button. If you instead use "Squash and merge," you'll end up with one commit
-that "squashes" together the nicely-curated logical changes and messages from
-the PR's constituent commits. If your PR has fixup commits to address review
-feedback, or generally sloppy commits, you should rebase those into a more
-logical commit series locally and force-push.
+revert at least one PR that you land in your first three months.
+
+You shouldn't merge yourself, instead, request our build bot Craig to do it for
```

Comment:
suggest different wording: "Finally, to merge your PR, request a merge from our built bot Craig. Craig is ..."

Reply:
er, s/built/build/

Reply:
made a few edits based on your suggestion, trying to work it better into the surrounding context, PTAL